setup.py

```diff
@@ -144,6 +144,6 @@ def get_versions(default={}, verbose=False):
         # needed for leap.common.http
         #  service_identity needed for propper hostname identification,
         #  see http://twistedmatrix.com/documents/current/core/howto/ssl.html
-        'Twisted': ["Twisted>=14.0.2", "service_identity", "zope.interface"]
+        'http': ["Twisted>=14.0.2", "service_identity", "zope.interface"]
```

Comment:
is this a hard dependency ? because twisted is only backported to wheezy in version 13.2.0-1~bpo70+1.